config/forms/event_details.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" ?>
+<form xmlns="http://schemas.sulu.io/template/template"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/form-1.0.xsd"
+>
+    <key>event_details</key>
+
+    <properties>
```

Comment:
The form view can be structured by using `<section>` tags in the form configuration. An example can be found in the [form configuration of the built-in contact entity](https://github.com/sulu/sulu/blob/2.x/src/Sulu/Bundle/ContactBundle/Resources/config/forms/contact_details.xml).